src/mudlet.cpp

```diff
@@ -3902,7 +3902,7 @@ void mudlet::createMapper(bool loadDefaultMap)
                                      pMap->mPlayerRoomOuterColor,
                                      pMap->mPlayerRoomInnerColor);
 
-    pMap->mpMapper = new dlgMapper(pHost->mpDockableMapWidget, pHost, pMap); //FIXME: mpHost definieren
```

Comment:
Again this is dropped without replacement?

Reply:
Yes it is. There is no "mpHost" to be defined here. The comment is ten+ years old and has survived three refactorings by now.

Reply:
Fair enough.